py/server/deephaven/experimental/s3.py

```diff
@@ -51,7 +51,8 @@ def __init__(self,
             region_name (str): the region name for reading parquet files. If not provided, the default region will be
                 picked by the AWS SDK from 'aws.region' system property, "AWS_REGION" environment variable, the
                 {user.home}/.aws/credentials or {user.home}/.aws/config files, or from EC2 metadata service, if running
-                in EC2.
+                in EC2. If no region name is derived from the above chain or the region name derived is incorrect for the
+            bucket accessed, the correct region name will be derived internally, at the cost of one additional request.
```

Comment:
does this indentation break pydocs? Not sure.

Reply:
It may.

Reply:
Will check.